src/inputs/NumberField.tsx

```diff
@@ -48,6 +48,8 @@ export interface NumberFieldProps {
   hideErrorMessage?: boolean;
   // Typically used for compact fields in a table. Removes border and uses an box-shadow for focus behavior
   borderless?: boolean;
+  inlineLabel?: boolean;
```

Comment:
Question: As a lazy ask, I assume this `inlineLabel` gets passed through `otherProps` to something else? (Just don't see it used here directly)

Reply:
@stephenh - Yes, the `inlineLabel` would get passed in through `otherProps` here.